app/routes/index.js

```diff
@@ -9,6 +9,9 @@ const app = choo();
 
 function body(template) {
   return function(state, emit) {
+    if (state.route.startsWith('/unsupported/')) {
+      state.promo = false;
```

Comment:
This will toggle the banner off for the remainder of the session, which for almost all cases is still fine because folks who end up here aren't likely to go to any other routes, but we can be more precise, so we should be.

I suggest creating a helper function called `showBanner` (or something) that combines your comparison here with stuff on line 16 below

Reply:
Ok, sure I'll do this